### ISSUE TITLE
Increase number of results in People Search

### ISSUE THREAD

**Issue by daffhunt** [bluesky-social/social-app#6656]
### Describe the Feature

The People tab of search only shows me a maximum of 25 accounts, with seemingly no way to scroll or page to see more. I assume this is a deliberate decision but it makes people search less useful. Especially as it always favours showing accounts that I follow. So if I am looking for people to follow with a shared interest (by searching for a word or hashtag in bios) once I have followed 25 such accounts, people search is no longer usable for finding more of them

### Describe Alternatives

- Increase length of results list of people search or scroll/paginate
- Allow search to be filtered to show only people followed/not followed

**Comment by LoganDupont** [comment]
This one is a huge problem as it's blocking the growth of Bluesky. Because of this people aren't found on Bluesky which is the reason why a lot of accounts are having problems to attract followers and leave the platform again. I'm for example an Angular developer and if I search on "angular" I don't see certain accounts ex. angularAir, angular connect, .... because the list is limited to 25 accounts. 

As we can see on https://www.bskycheck.com/stats.php the growth is slowing down and this is for sure one of the reasons because people can't seem to find the persons based on search results. @pfrazee & @mozzius can you have a look at this with high priority?

**Comment by LoganDupont** [comment]
@pfrazee & @mozzius after some more investigation I would even think it's a bug. If we take a look at https://github.com/bluesky-social/social-app/blob/077fd843f0d854a07d2cee5dfc3980dbb91db86d/src/screens/Search/SearchResults.tsx#L259 we can see that we don't check if we reach the end in order to a callback to fetch data on the next page.

**Comment by Remustarded** [comment]
Not just people but also feeds. As a platform heavily emphasizing on custom feeds Bluesky does a terrible job promoting them.

It seems to me whoever is steering Bluesky wants a quick explosive growth. They cater to TitTok users (video tab) and sports fans (official starter packs for drafting season BTW), instead of improving basic functionality like search and moderation.

Thanks developers like https://github.com/LoganDupont for making Bluesky experience better.

**Comment by LoganDupont** [comment]
@Remustarded Indeed, I already created some issues (#8293, #8301 & #8302) to track them. Probably this is also the case on other screens. I saw @mozzius ran the github actions yesterday so I hope this will be fixed very soon. @haileyok also mentioned this bug 8 months ago on bsky https://bsky.app/profile/hailey.at/post/3kzwqlcdeis2c. But I'm not sure why it hasn't been tackled in the meantime. I will try to have a look at the feeds as wel and maybe fix it.

**Comment by mozzius** [comment]
I appreciate your enthusiasm, but please can you refrain from tagging me and my collegues excessively. This issue is on our radar, and I'll get around for it - however, tagging us a bunch won't make that happen any faster. I've brought it up internally and we'll likely wrap it into some other search-related work that needs to be done.

**Comment by LoganDupont** [comment]
Okay no problem
